dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -106,13 +139,25 @@ public static void registerZenodoDOIForWorkflow(String zenodoUrl, String docksto
                 // accessed through the "latest_draft" under "links" in the response body.
                 Object links = returnDeposit.getLinks();
                 String depositURL = (String)((LinkedHashMap)links).get("latest_draft");
-
                 String depositionIDStr = depositURL.substring(depositURL.lastIndexOf("/") + 1).trim();
                 // Get the deposit object for the new workflow version DOI
                 depositionID = Integer.parseInt(depositionIDStr);
                 returnDeposit = depositApi.getDeposit(depositionID);
 
                 depositMetadata = returnDeposit.getMetadata();
+
+                // Retrieve the DOI so we can use it to create a Dockstore alias
+                // to the workflow; we will add that alias as a Zenodo related identifier
+                String doi = depositMetadata.getDoi();
+                String alias = createAliasUsingDoi(doi, workflow);
+
+                // Add the new alias created with the DOI to the deposit metadata
+                // Later on we will update the Zenodo deposit (put the deposit on
```

Comment:
Where do you do this? Could you please mention the method, I'm having a hard time figuring it out.

Reply:
Added to comment